README.md

```diff
@@ -55,18 +55,20 @@ Create a new file in `src/test/scala/`, for example, `BasicTest.scala`.
 In this file:
 1.  Add the necessary imports:
     ```scala
-    import org.scalatest._
-    import chiseltest._
-    import chisel3._
+   import org.scalatest._
+   import org.scalatest.flatspec._ 
+   import matchers.should._
+   import chiseltest._
+   import chisel3._
```

Comment:
Scalafix organize-imports usually formats this as:

```
import chisel3._
import chiseltest._
import org.scalatest._

import flatspec._
import matchers._
```

I believe it's more clear.

Reply:
Personally I would prefer just:
```
import chisel3._
import chiseltest._
import org.scalatest.flatspec.AnyFlatSpec
```

Reply:
I like the minimal imports, and what about `Matchers`?

Reply:
The whole `Matchers` thing is mostly a matter of taste...

If you do not have `Matchers`, you can still use regular asserts to perform checks in your tests + the `expect` function from chiseltest also provides you with a built-in assert. I personally believe that for new comers writing asserts is less confusing and if people want to they can always import `Matchers`, I just prefer not to include it by default.

In case this was confusing:
```.scala
// without matchers
assert(foo() == 42)
// with matchers
foo() should be (42)
```